uint8 did_reach_first_goal
uint8 did_reach_second_goal
float32 pick_x
float32 pick_y
float32 drop_x
float32 drop_y

